docker/nl/docker-compose.yaml

```
@@ -20,20 +20,20 @@ version: "3.8"
 services:
   database:
     build:
-      context: source/database
+      context: ../common/database
       args:
-        - DATABASE_VERSION=${IMAGE_TAG}_${GITHASH}
-    image: ${REGISTRY_URL}emission-database:${IMAGE_TAG}
+        - DATABASE_VERSION=${AERIUS_IMAGE_TAG}_${AERIUS_GITHASH}
+    image: ${AERIUS_REGISTRY_URL}database:${AERIUS_IMAGE_TAG}
     restart: unless-stopped
     volumes:
       - /etc/localtime:/etc/localtime:ro
```

```suggestion
    image: ${AERIUS_REGISTRY_URL}database:${AERIUS_IMAGE_TAG}
    restart: unless-stopped
    volumes:
      - /etc/localtime:/etc/localtime:ro
    environment:
      - POSTGRES_USER=${AERIUS_DATABASE_USERNAME}
      - POSTGRES_PASSWORD=${AERIUS_DATABASE_PASSWORD}
```
Apparently we don't do this properly for `calculator`, but we do for `register`

---

Difference here might be that calculator uses our own database-build image as a basis, while register uses the generic postgres image. So for Register it's actually required, while for calculator you can get away with using the defaults...

Emission service is more like Calculator (uses our own image as a basis), but guess it can't hurt to apply this.